gobblin-api/src/main/java/gobblin/configuration/ConfigurationKeys.java

```diff
@@ -161,6 +161,7 @@
   public static final String EXTRACT_TABLE_TYPE_KEY = "extract.table.type";
   public static final String EXTRACT_NAMESPACE_NAME_KEY = "extract.namespace";
   public static final String EXTRACT_TABLE_NAME_KEY = "extract.table.name";
```

Comment:
What is the difference between entity and table? QueryBasedSource seems to be using those interchangeably? Let's agree on terminology and stick to it.